content/articles/statistical-hypothesis-testing-python-implementation/index.md

```diff
@@ -0,0 +1,506 @@
+---
+layout: engineering-education
+status: publish
+published: true 
+url: /statistical-hypothesis-testing-python-implementation/
+title: Statistical Hypothesis Testing With Python Implementation
+description: This tutorial is a comprehensive explanation and implementation of statistical hypothesis testing methods such as T-Test, Chi-square test & ANOVA in Python.
+author: qoyum-olatunde-yusuf
+date: 2021-06-16T00:00:00-11:00
+topics: [Statistical Analysis, Hypothesis Testing, Data Science, Python ]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/statistical-hypothesis-testing-python-implementation/hero.PNG
+ alt: statistic hypothesis testing with python cover image
+ 
+---
+Hypothesis is the act of making observations and generating probing questions from the observations made in an attempt to come to a conclusion. Hypothesis testing is the test of assumptions on a population sample. It involves checking out whether a hypothesis should be accepted or not. In the business world and in real life, hypothesis testing has been of great importance in making worthy decisions off mere observations. Billions of data are being generated on a daily basis around the globe and this has made the relevance of hypothesis testing a stronghold in making business decisions.
+<!--more-->
+### Table of Contents
+1. Prerequisites.
+2. Goals of the tutorial.
+3. Understanding hypothesis testing.
+4. Confidence of null hypothesis.
+5. Statistical techniques for hypothesis testing.
+6. Chi-square test - Python implementation.
+7. T-Test - Python implementation.
+8. ANOVA - Python implementation.
+9. Conclusion.
+10. Additional resources.
```

Comment:
@wizardcalidad Please take a look at other's drafts. I suggest you look through [this draft](https://raw.githubusercontent.com/section-engineering-education/engineering-education/master/content/articles/boosting-algorithms-python/index.md) to see how this section is formatted.
And please edit the other headings in the same way.
Like I've mentioned, please take the time to read through some drafts of fellow publishers to understand the format.